syncall/types.py

```diff
@@ -40,6 +40,73 @@ class TaskwarriorRawItem(TypedDict, total=False):
 # Item as returned from the Taskw Python API on tw.get_task(id=...)
 TaskwarriorRawTuple = Tuple[Optional[int], TaskwarriorRawItem]
 
+# ---------------------------------------------------------------------------------------------
+# Google Tasks
+
+
+class GTaskLink(TypedDict):
+    description: str
+    link: str
+    type: str  # "email"
+
+
+class GTasksList(TypedDict):
+    etag: str  # ETag of the resource.
+    id: str  # Task list identifier.
+    kind: str  # Type of the resource. This is always "tasks#taskList".
+    selfLink: str  # URL pointing to this task list. Used to retrieve, update, or delete this task list.
+    title: str  # Title of the task list.
+    updated: str  # Last modification time of the task list (as a RFC 3339 timestamp).
+
+
+class GTasksItem(TypedDict):
+    """Dictionary part of an item as returned from the Google Tasks Python API on tasks().get_task()
+
+    Example:
+
+       {
+        'id': 'Yl9GSzNDVWluTk9heE1sUQ',
+        'kind': 'tasks#task',
+        'status': 'completed',
+        'etag': '"LTc5ODEwNzk2Mg"',
+        'title': 'Simple completed item',
+        'updated': '2021-12-04T15:07:00.000Z',
+        'selfLink': 'https://www.googleapis.com/tasks/v1/lists/YUFLWXdFQ3NLczVKalZsWg/tasks/Yl9GSzNDVWluTk9heE1sUQ',
+        'position': '00000000000000000001',
+        'notes': '''
+            * Annotation: Testing adding annotation in Google Tasks\n
+
+            * status: done\n
+            * uuid: 542f5dbc-b1b7-4a85-b55e-10f5f1d31847
+        ''',
+        'due': '2021-12-04T18:07:00.000Z',
+        "completed": "2021-12-04T15:07:00.000Z",
+        'links': []
+    }"""
+
+    completed: Optional[
+        str
+    ]  # Completion date of the task (as a RFC 3339 timestamp). This field is omitted if the task has not been completed.
```

Comment:
nitpick: Could you put the comment at the previous line from the variable? I find it to be a bit easier to read that way. i.e.:


    # Completion date of the task (as a RFC 3339 timestamp). This field is omitted if the task has not been completed.
    completed: Optional[str]

Reply:
done